docs/getting-started.md

```diff
@@ -0,0 +1,76 @@
+
+
+## Usage
+
+Dolphin has a main command line entry point to run the algorithms and tools in workflows.
+The main entry point is named `dolphin`, which has two subcommands:
+
+1. `dolphin config`: create a workflow configuration file.
+2. `dolphin run` : run the workflow using this file.
+
+Example usage:
+
+```bash
+$ dolphin config --slc-directory /path/to/slc --ext ".tif"
+```
+This will create a YAML file (by default `dolphin_config.yaml` in the current directory).
+You can also directly use a list of SLC files as input, e.g.:
+```bash
+$ dolphin config --slc-files /path/to/slc1.tif /path/to/slc2.tif
+```
+
+
+
+## Install
```

Comment:
My thinking was this (similar what some projects do, e.g. https://github.com/rasterio/rasterio#rasterio ):
When the first thing you see is a simple example, it can act as an advertisement- "look how easy this is to use!". 
Then after you see that, you can decide if it's worth going to install it. But I often like to see the simplest use case as the first thing I see in a README before detailed installation instructions with various platforms/environments/caveats.

I'm not super attached to the ordering, but I do occasionally see the simple example up front to hook you.

Reply:
Oh OK I see the logic. I don't think I've noticed that approach before in other projects' docs pages, but it makes sense :+1: